elixir/apps/web/test/web/live/resources/new_test.exs

```diff
@@ -204,6 +237,13 @@ defmodule Web.Live.Resources.NewTest do
       connections: %{connection.gateway_group_id => %{enabled: false}}
     }
 
+    # TODO: BETA CHANGE Use that attrs above this after beta
```

Comment:
I think we should not stop testing features even if we don't give them to our users, otherwise, we can introduce breaking changes that we will not be aware of or later they will simply not work again when we uncomment this code.

So let's have tests cover all the functionality, but we need to add a `hide_beta_features` environment variable and just add if's to hide forms in the UI when it's set to true. You will be also able to write tests for behavior changes depending on the flag value.

Reply:
I agree, that would be the ideal state.  Perhaps I'm misinterpreting what you're saying, but it would essentially be feature flags correct?  If so, I thought we were not planning on doing that for a while?

I did this as a quick way to get these "features" disabled in the UI, with what was hopefully a relatively easy way to identify and rollback the changes here when we were ready (which I was thinking/hoping would be a short time frame).  I'm definitely happy to do whatever the team feels is best though.

Reply:
+1 to what @AndrewDryga said, but can we do it per-account instead? We want to show features for our own testing on staging and prod but hide them for everyone else.

Reply:
@jamilbk, this can definitely be done, and I feel like what you're describing is a full on feature flag system.  Happy to start working on that if a feature flag system is needed.

Reply:
@bmanifold Hmm yeah, I suppose. In my head I was thinking we could do a poor man's feature flagging system for now:

- Have a whitelist of account slugs to enable for: e.g. `canary_accounts`
- Add helper to check if `@account` if in this whitelist and hide/show based on that
- Requires a PR and deploy to change the whitelist

I don't think we need a remote feature flagging system just yet, but I might not be thinking this through all the way

Reply:
Did not want us to have per-account feature flags (and have just one ENV variable for now) but I see where Jamil is coming from 🙃. Let's do the most simple version of this possible: 

- [ ] Add a `:feature_flags` env config (by adding `feature_flags: [self_hosted_relays: [account_ids..]]`-like config to `definitions.ex`, `config.exs` and `release.exs`).
- [ ] Add `is_feature_enabled?(account_id, :self_hosted_relays)` to `domain/config.ex` which will check if the config key is set. This feature then can be used pretty much anywhere and overridden with `Domain.Config.put_env_override/1`
- [ ] Write tests that make sure that, for example,  if sign-up is disabled then the form is not visible. But keep the old tests working.

This will make enabling those features a deployment action (to change the ENV) but I really hope that we won't need to move it to the database and spread DB queries across application code to assert on them.

Reply:
Ok, I'll give this a shot.  I was a little nervous that I might be pulling at a thread that never ended if I did a feature flag system, but I'll give this a go.